### PR TITLE
OP-990 Add further check for underlying data changes in billing

### PR DESCRIPTION
**Pull request by mwithi**
To complete OP-990 work.

### CODE REVIEW

File: src/main/java/org/isf/accounting/gui/PatientBillEdit.java [hunk 1]
```diff
@@ -570,6 +578,11 @@ private void checkBill() {
 						modified = true;
 					}
 				}
+				if (!thisBill.getAdmission().getWard().equals(currentAdmission.getWard())) {
+					MessageDialog.info(PatientBillEdit.this,
+									MessageBundle.formatMessage("angal.newbill.thepatienthasbeentransferredmeanwhilearrow.fmt.msg",
```
Comment (dbmalkovsky) [inline src/main/java/org/isf/accounting/gui/PatientBillEdit.java:583]:
```suggestion
									MessageBundle.formatMessage("angal.newbill.thepatienthaschangedwardsarrow.fmt.msg",
```

**Review by dbmalkovsky — CHANGES_REQUESTED** [review]
some minor wording improvements

File: bundle/language_en.properties [hunk 2]
```diff
@@ -1166,7 +1166,9 @@ angal.newbill.somepriceshavebeenchangeddoyouwanttoupdatetheitemsprices.fmg.msg
 angal.newbill.somepricesnotfound.fmt.msg                                                               = Some prices not found: {0}
 angal.newbill.somepricesnotfoundandsomeotherchangeddoyouwanttoupdatetheitemsprices.fmt.msg             = Some prices not found and some other changed:\nNot found: {0}\nChanged: {1}\n\nDo you want to update the existing items' prices?
 angal.newbill.thedateisbeforethelastpayment.msg                                                        = The date is before the last payment.
+angal.newbill.thepatienthaschangedwardsarrow.fmt.msg                                                                  = The patient has changed wards: {0} -> {1}
```
Comment (dbmalkovsky) [inline bundle/language_en.properties:1169]: format?

Comment (mwithi) [reply bundle/language_en.properties]: fixed. I think was coming from your suggestino

**Pull request merged**